business_requirement_deliverable_cost/models/business.py

```diff
@@ -73,46 +73,25 @@ def product_uom_change(self):
                 resource.sale_price_unit = product.price
 
 
-class BusinessRequirementDeliverable(models.Model):
-    _inherit = "business.requirement.deliverable"
-
-    unit_price = fields.Float(
-        groups='business_requirement_deliverable_cost.\
-group_business_requirement_estimation',
-    )
-    price_total = fields.Float(
-        groups='business_requirement_deliverable_cost.\
-group_business_requirement_estimation',
-    )
-
-
 class BusinessRequirement(models.Model):
     _inherit = "business.requirement"
 
     total_revenue = fields.Float(
         store=True,
-        groups='business_requirement_deliverable_cost.\
-group_business_requirement_estimation',
```

Comment:
Why remove the ACL here? 
Adding ACL in forms looks good to me but not sure we should remove here.
@duanyp1991